### PR TITLE
Section 2

### PR DESCRIPTION
**Pull request by katsu223**
[Deploy link](https://katsu223.surge.sh)

### CODE REVIEW

**Comment by katsu223** [comment]
katsu223.surge.sh

File: index.html [hunk 1]
```diff
@@ -13,63 +13,74 @@
...
+        <img class="divider" src="resources/images/background/divider.png">
+        <br>
+        <div class="button-group">
+          <a class="button button-book" href="#book">book a table</a>
+          <a class="button button-menu" href="#menu">see the menu</a>
+        </div>
+      </div>
     </div>
+      <section class="first-section">
+        <div class="flex-box">
```
Comment (magicsd) [inline index.html]: Class name should be more descriptive

File: resources/css/index.css [hunk 2]
```diff
@@ -162,3 +162,43 @@ body, html, ul, ol, li, p, h1, h2, h3, h4 {
   font-size: 90px;
   text-shadow:  1px 1px 2px #000;
 }
+
+.first-section{
+  background:  url("../images/background/pattern.png");
+  background-position: 50% 0px;
+  box-shadow: inset 0px 0px 102px 6px rgba(0, 0, 0, .1);
```
Comment (magicsd) [inline resources/css/index.css]:
```suggestion
  box-shadow: inset 0 0 102px 6px rgba(0, 0, 0, .1);
```

File: index.html (hunk 1 above)
Comment (katsu223) [reply index.html]: fixed

File: resources/css/index.css (hunk 2 above)
Comment (katsu223) [reply resources/css/index.css]: fixed

**Pull request merged**